src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private Movie movie;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private CinemaHall cinemaHall;
+
+    private LocalDateTime showTime;
```

Comment:
The `showTime` field is missing access modifiers. It should be private to follow the encapsulation principle.